torch/csrc/jit/script/module.h

```diff
@@ -469,22 +469,24 @@ struct Module {
 
   void save(const std::string& filename);
 
-  void copy_into(std::function<std::shared_ptr<Module>(std::vector<std::string>)> module_lookup, std::vector<std::string> names = {}) const {
-    std::unordered_map<at::Tensor*, at::Tensor*> parameter_remap;
+  void copy_into(std::function<std::shared_ptr<Module>(
+      std::vector<std::string>)> module_lookup,
+      std::unordered_map<at::Tensor*, at::Tensor*>& parameter_remap,
+      std::vector<std::string> names = {}) const {
     auto curr = module_lookup(names);
     for (auto &kv : parameters) {
       curr->register_parameter(kv.key(), *kv.value().slot(), kv.value().is_buffer);
       parameter_remap[kv.value().slot()] = curr->parameter_slot(kv.key());
     }
     for (auto &kv : modules) {
       names.push_back(kv.key());
-      kv.value().module->copy_into(module_lookup, names);
+      kv.value().module->copy_into(module_lookup, parameter_remap, names);
       names.pop_back();
     }
     for (auto &kv : methods) {
```

Comment:
Comment: Submodules must be translated first, otherwise parameter_remap entries will not be filled in for methods of this module.